src/ui/widgets/editors/w_RoutesEditor.cpp

```diff
@@ -299,9 +299,14 @@ CONFIGROOT RouteEditor::OpenEditor()
     {
         // Process Browser Forwarder
         QJsonObject browserForwarder;
-        browserForwarder["listenAddr"] = bfListenIPTxt->text();
-        browserForwarder["listenPort"] = bfListenPortTxt->value();
-        root["browserForwarder"] = browserForwarder;
+		if(bfListenIPTxt->text().trimmed() == ""){
+			root["browserForwarder"] = QJsonValue::Null;
```

Comment:
What if we don't set it at all?

Reply:
In fact I don't know what will happen. So I set it null.